_posts/en/newsletters/2022-12-07-newsletter.md

```diff
@@ -55,12 +55,96 @@ infrastructure projects.
 meeting, highlighting some of the important questions and answers.  Click on a
 question below to see a summary of the answer from the meeting.*
 
-FIXME:larryruane
+[Bump unconfirmed ancestor transactions to target feerate][review club 26152-2]
+is a PR by Xekyo (Murch) that improves the accuracy of the wallet's fee
+calculation in the case that unconfirmed UTXOs are selected as inputs.
```

Comment:
We could also point out that we already estimate the correct fees during coin selection which allows us to correctly deprioritize use of unconfirmed UTXOs that require a higher bump fee.